src/messages.lisp

```diff
@@ -125,6 +145,12 @@
       :documentation "The RPC request id."
       :type :string
       :required t)
+
+     (|warnings|
```

Comment:
What is a use case for errors also having warnings?

Reply:
I can imagine code being written in a way that it raises some warnings along the lines of "This seems dangerous, but I'm going to try to proceed" before ultimately hitting an unrecoverable error. Someone debugging such a call might like to hear about those warnings.